lib/compiler_rt/fixsfdi.zig

```diff
@@ -8,6 +9,10 @@ comptime {
         @export(&__aeabi_f2lz, .{ .name = "__aeabi_f2lz", .linkage = common.linkage, .visibility = common.visibility });
     } else {
         @export(&__fixsfdi, .{ .name = "__fixsfdi", .linkage = common.linkage, .visibility = common.visibility });
+
+        if (builtin.cpu.arch.isArmOrThumb() and builtin.target.isMinGW()) {
```

Comment:
could you please put this logic (`builtin.cpu.arch.isArmOrThumb() and builtin.target.isMinGW()`) into common.zig? This will prevent future contributors from missing some sites if the logic needs to be adjusted in the future.

Reply:
Done.

Reply:
Thanks!